### PR TITLE
Add "has biography" people search filter 

### PR DESCRIPTION
**Pull request by maradragan**
This makes it possible to search for people with biography filled in or with an empty biography.

Implements #2305 

- [ ] needs migration through java command clearEmptyBiographies

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -423,6 +423,17 @@ task integrationTest(type: Test) {
 	environment(run.environment)
 }
 
+// Create the task that runs the migration script for empty HTML biography fields of persons
+task dbClearBiographies(dependsOn: 'compileJava', type: JavaExec) {
```
Comment (VassilIordanov) [inline build.gradle]: I suggest to rename to dbClearEmptyBiographies as current name is misleading

Comment (VassilIordanov) [inline build.gradle]: Isn't this better invoked as a `customChange` through liquibase?

Comment (maradragan) [reply build.gradle]: I agree, changed that.

Comment (maradragan) [reply build.gradle]: I can have a look at that next week.

Comment (gjvoosten) [reply build.gradle]: Liquibase's options are too limited (we need access to a lot of the classes).

**Pull request merged**